pom.xml

```diff
@@ -6,7 +6,7 @@
     <parent>
         <groupId>org.icgc_argo</groupId>
         <artifactId>workflow-graph-parent-pom</artifactId>
-        <version>1.12.0</version>
+        <version>1.13.0</version>
```

Comment:
this version doesn't exist yet but it will after workflow-graph-lib is released and I update the parent POM repo.